src/backend/parser/parse_utilcmd.c

```diff
@@ -2115,14 +2115,14 @@ transformCreateExternalStmt(CreateExternalStmt *stmt, const char *queryString)
 		{
 			DefElem    *defel = (DefElem *) lfirst(exec_location_opt);
 
-			if (strcmp(defel->defname, "master") == 0)
+			if (strcmp(defel->defname, "coordinator") == 0)
```

Comment:
```suggestion
			if (strcmp(defel->defname, "coordinator") == 0 || strcmp(defel->defname, "master") == 0)
```

Bonus point, we can emit a deprecation warning if the user spelled it as "master"